### DISCUSSION TITLE
考虑取消 select组件下拉 里的✅,  增加 desc 的支持吗?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#42768] [General]
单选 下拉里无✅
<img width="241" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/4013cb7b-cd5e-46a8-9287-a13602843dd9">

多选,下拉里有✅
<img width="520" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/3bc4154d-3e72-4d56-83ea-f87720c02c2b">

首先,这就不统一, 其次,这个✅不仅没有意义, 还占用了很重要的 desc 属性, 参考下 element-plus
<img width="836" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/52b6c2d4-0054-4367-9696-4dbc42af3e45">

**Comment by afc163** [comment]
多选没有 ✅ 选中项的识别性会差很多。

**Reply by vaynevayne** [reply]
核心是增加 desc, 但是 desc 只能放右边, ✅占用了这个位置, 如果非要✅,那也不能放右边

**Reply by yoyo837** [reply]
还有RTL

**Reply by vaynevayne** [reply]
我觉得还是取消✅吧, 有时候要定制 option, 左边要放 icon

**Comment by MadCcc** [comment]
之前有讨论过，类似 mac 原生的这种吧
![image](https://github.com/ant-design/ant-design/assets/27722486/a3c73c7c-bbb5-4a74-8b01-78271880f700)

**Reply by vaynevayne** [reply]
good
